vuepress/docs/next/tutorials/compose/content-tutorial.md

```diff
@@ -1,108 +1,89 @@
 
-![image](./img/Content3.png)
+![Add Content Information](./img/content-addinfo.png)
 
-Save the new content by selecting:
+## Saving and Approving Content 
+1. For new or edited content, determine which `Status` it belongs to:  
+`Status`:
+ * `Draft`: The content is in the development stage and not ready for approval or publication. The `Save` or `Save and Continue` buttons establish this status for all content.
 
--   **Save**: The content will be saved in a draft version.
+ * `Ready`: The content is ready for review and has not yet been approved nor published.  Items with this status can be saved and approved later or published directly.
 
--   **Save and Continue**: The content will be saved and the editable form will be displayed.
+2. Saving options:
 
--   **Save and Approve**: The content will be saved, approved and published.
+-   **Save**: The content will be saved in a draft version.
 
-## Finding Content
+-   **Save and Continue**: The content will be saved as a draft version and the editable form will be displayed.
 
-You can list specific Content by using the filters:
+-   **Save and Approve**: The content will be saved, approved, and ready for use in an application.
 
--   **All**: All created Content is displayed (Draft, To Approve and
-    Approved/Published)
 
--   **Pending Changes**: All created Content that was saved by only clicking on the
-    **Save** or **Save and Continue** button. The content is not
-    approved and therefore is not ready to be published.
+## Finding Content
 
--   **Ready for Approval**: All created Content that is in **Ready** status,
-    and saved by clicking on the **Save** or **Save and Continue** button.
-    The content is ready for review but has not yet been approved and
-    therefore has not been published.
+Using the `Advanced Filters` feature, you can search for Content by name, type, category, group and status. \
+![Content Filters](./img/content-filters.png)
 
--   **Published**: All created Content that has been saved and approved
-    by clicking on the **Save and Approve** button. This content has been
-    published.
+In the resulting table, contents are listed with these parameters: `Name`, `Author`
+`Code`, `Created by`, `Last Edited`, `Type`, `Status`, `Visibility`, `Group`, `Creation` Date, `Last Modified`, and executable `Actions`. Many of the fields can be deleted from view by unchecking them in the `Additions to the Table of Results` drop-down menu.
 
-![image](./img/Content5.png)
+![Content Table List View](./img/content-table-view.png)
 
-In the table list, contents are listed with several parameters: Name,
-Created by, Last Edited, Type, Created Date, Owner Group, Join Group,
-Status, Restrictions, Code, Actions. Fields can be added or deleted from view by selecting or unselecting them
-in the Columns drop-down menu.
+## Content Actions
 
-![image](./img/Content6.png)
+For all items in the `Content` table, the `Actions` drop-down menu provides these executable options:
```

Comment:
these -> the following (that's what we usually use so is consistent; also a bit more specific/formal)

Reply:
I don't think we need to be that exacting, when meaning is not impaired or it's uncomplicated.